File: KSCIRC.Models/Model/StatValue.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace KSCIRC.Models.Model
{
    public partial class StatValue
    {
        public int Id { get; set; }
        public int GeneId { get; set; }
        public int PublicationId { get; set; }
        public int? DaysPostInjury { get; set; }
        public decimal? InputValue { get; set; }
        public decimal? InputQvalue { get; set; }
        public decimal? ImmunoprecipitateValue { get; set; }
        public decimal? ImmunoprecipitateQvalue { get; set; }
        public decimal? EnrichmentValue { get; set; }
        public decimal? EnrichmentQvalue { get; set; }
        public decimal? InteractionValue { get; set; }
        public decimal? InteractionQvalue { get; set; }

        public virtual Gene Gene { get; set; }
        public virtual Publication Publication { get; set; }
    }
}

File: Web/src/app/models/statValue.ts
export class StatValue {
    Id?: number;
    DaysPostInjury?: number;
    InputValue?: number;
    InputQValue?: number;
    ImmunoprecipitateValue?: number;
    ImmunoprecipitateQValue?: number;
    EnrichmentValue?: number;
    EnrichmentQValue?: number;
    InteractionValue?: number;
    InteractionQValue?: number;

    constructor(
        id: number = 0,
        daysPostInjury: number = 0,
        inputValue: number = 0,
        inputQValue: number = 0,
        immunoprecipitateValue: number = 0,
        immunoprecipitateQValue: number = 0,
        enrichmentValue: number = 0,
        enrichmentQValue: number = 0,
        interactionValue: number = 0,
        interactionQValue: number = 0
    ) {
        this.Id = id,
        this.DaysPostInjury = daysPostInjury,
        this.InputValue = inputValue,
        this.InputQValue = inputQValue,
        this.ImmunoprecipitateValue = immunoprecipitateValue,
        this.ImmunoprecipitateQValue = immunoprecipitateQValue,
        this.EnrichmentValue = enrichmentValue,
        this.EnrichmentQValue = enrichmentQValue,
        this.InteractionValue = interactionValue,
        this.InteractionQValue = interactionQValue
    }
}